ci-operator/step-registry/single-node/recert/single-node-recert-commands.sh

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Echo a script to be run as our systemd unit to a file so we can base64 encode it.
+recert_script=$(cat << 'EOF'
+#!/usr/bin/env bash
+
+set -euoE pipefail
+
+export KUBECONFIG=/etc/kubernetes/static-pod-resources/kube-apiserver-certs/secrets/node-kubeconfigs/localhost.kubeconfig
+function wait_for_api {
+  echo "Waiting for API..."
+  until oc get clusterversion &> /dev/null
+  do
+    echo "Waiting for API..."
+    sleep 5
+  done
+  echo "API is available"
+}
+
+function start_containers {
+  systemctl start crio
+  systemctl start kubelet
+}
+
+function stop_containers {
+  systemctl stop kubelet
+  crictl ps -q | xargs crictl stop || true
+  systemctl stop crio
+}
+
+function recert {
+  RELEASE_IMAGE=quay.io/openshift-release-dev/ocp-release:4.13.6-x86_64
```

Comment:
```suggestion
  RELEASE_IMAGE=quay.io/openshift-release-dev/ocp-release:4.13.6-x86_64
```

We need to steal the release image from the cluster we're running on, instead of hardcoding it

Reply:
Absolutely, please check my comment [here](https://github.com/openshift/release/pull/43149#discussion_r1324550671), which addresses this as well as the `recert` image.

Reply:
For the release image, maybe it's best to just take it from the etcd pod. This would make this script less dependent on prow